tutorial/cpp/Makefile

```diff
@@ -20,8 +20,8 @@ gpu: $(GPU_TARGETS)
 $(CPU_TARGETS): %: %.cpp ../../libfaiss.a
 	$(CXX) $(CXXFLAGS) -o $@ $^ $(LDFLAGS) -I../../.. $(LIBS)
 
-$(GPU_TARGETS): %: %.cpp ../../libfaiss.a ../../gpu/libgpufaiss.a
-	$(NVCC) $(NVCCFLAGS) -o $@ $^ $(NVCCLDFLAGS) -I../../.. $(NVCCLIBS)
+$(GPU_TARGETS): %: %.cpp ../../libfaiss.a
+	$(CXX) $(CXXFLAGS) -o $@ $^ $(LDFLAGS) -I../../.. $(LIBS)
```

Comment:
Now that both rules are the same, no need for static patterns: you can replace both `$(CPU_TARGETS): %: %.cpp ../../libfaiss.a` and `$(GPU_TARGETS): %: %.cpp ../../libfaiss.a` rules with one generic rule:
```
%: %.cpp ../../libfaiss.a
	$(CXX) $(CXXFLAGS) -o $@ $^ $(LDFLAGS) -I../../.. $(LIBS)
```

Reply:
I didn't realize that is all I need. I guess still have a lot to learn about Makefile. \ o /